File: Frontend/src/app/core/models/notifications/notification.dto.ts
export interface NotificationDto {
    id: string;
    title: string;
    message: string;
    type: string;
    actorName: string;
    targetType: string;
    targetId: string;
    route: string;
    isRead: boolean;
    createdAt: string;
    expiresAt: string;
}

File: Backend/LogisticsPlatform.Application/DTOs/Notifications/NotificationDto.cs
namespace LogisticsPlatform.Application.DTOs.Notifications;

public class NotificationDto
{
    public Guid Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Message { get; set; } = string.Empty;
    public string Type { get; set; } = string.Empty;
    public string ActorName { get; set; } = string.Empty;
    public string TargetType { get; set; } = string.Empty;
    public Guid TargetId { get; set; }
    public string Route { get; set; } = string.Empty;
    public bool IsRead { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime ExpiresAt { get; set; }
}
